src/solvers/strings/format_specifier.cpp

```diff
@@ -27,8 +27,10 @@ static format_specifiert format_specifier_of_match(std::smatch &m)
   std::string flag = m[2].str();
   int width = m[3].str().empty() ? -1 : std::stoi(m[3].str());
   int precision = m[4].str().empty() ? -1 : std::stoi(m[4].str());
+  // `tT` values:  "t" = date/time, "T" = uppercase date/time
   std::string tT = m[5].str();
 
+  // date/time
   bool dt = !tT.empty();
   if(tT == "T")
     flag.push_back(format_specifiert::DATE_TIME_UPPER);
```

Comment:
it's strang we don't do anything in the "t" case

Reply:
date/time is not yet supported, so untested, so it may well be missing.